tests/introspection/ApiBaseTest.cs

```diff
@@ -202,6 +217,24 @@ public bool MemberHasObsolete (MemberInfo member)
 #endif
 		}
 
+		public bool MemberHasUnsupported (MemberInfo member)
+		{
+#if NET
+			return member.GetCustomAttributes<UnsupportedOSPlatformAttribute> (false).Any ();
+#else
+			return member.GetCustomAttribute<ObsoleteAttribute> () is not null;
+#endif
+		}
+
+		public bool MemberHasEditorBrowsableNever (MemberInfo member)
+		{
+			var attribute = member.GetCustomAttribute<EditorBrowsableAttribute> (false);
+			if (attribute is null)
+				return false;
+
+			return attribute.State == EditorBrowsableState.Never;
```

Comment:
If the attribute is null, it will never be == to EditorBrowsableState.Never so something like the following should do the trick:
```suggestion
			var attribute = member.GetCustomAttribute<EditorBrowsableAttribute> (false);
			return attribute?.State == EditorBrowsableState.Never;
```